lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/implementation/ConfigurationImpl.java

```diff
@@ -30,7 +30,10 @@
 import java.util.Set;
 
 public class ConfigurationImpl implements Configuration {
-  private final ConfigurationBuilderImpl builder;
+
+  private ConfigurationBuilderImpl builder=null;
+
+  public ConfigurationImpl(){}
```

Comment:
Do we need it? Where do we use it?

**P.S.** A side note: please tune your IDE to autoindent and format your code according to our [coding guidelines](https://cwiki.apache.org/confluence/display/REEF/Coding+Guidelines)

Reply:
Will do